starter_code/src/movies.js

```diff
@@ -4,9 +4,37 @@
 
 // Get the average of all rates with 2 decimals 
 
+var totalRates = movies.reduce(function (sum, movie){
```

Comment:
You should put this inside the ratesAverage function or in another fuction.

Reply:
If we change the data, the variable can't be modified, if you put it inside a function you can always call the function.